src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,40 @@
+import { FilterType } from '../../types/FilterType';
+import { Filter } from '../Filter/Filter';
+
+interface FooterProps {
+  filter: string,
+  filterSelected: (filter: FilterType) => void,
+  todosLength: number,
+  hasCompletedTodos: boolean,
+}
+
+export const Footer: React.FC<FooterProps> = ({
+  filter,
+  filterSelected,
+  todosLength,
+  hasCompletedTodos,
+}) => {
+  return (
+    !todosLength ? (
```

Comment:
You already have this condition on `App` component

Reply:
done